src/main/java/com/epam/reportportal/extension/monday/command/PostTicketCommand.java

```diff
@@ -48,7 +48,7 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import java.util.List;
 import java.util.Map;
-import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.collections.CollectionUtils;
```

Comment:
https://mvnrepository.com/artifact/commons-collections/commons-collections
commons-collections is the library from 2015. Newer one is collections4